frontend/src/api/ApiCaller.tsx

```diff
@@ -210,11 +210,12 @@ export class BackendAPICaller {
         return result.content
     }
 
-    static async postLocalizationMission(localizationPose: Pose, robotId: string) {
-        const path: string = 'robots/' + robotId + '/start-localization'
+    static async postLocalizationMission(localizationPose: Pose, robotId: string, deck: string) {
+        const path: string = 'robots/start-localization'
         const body = {
-            position: localizationPose.position,
-            orientation: localizationPose.orientation,
+            robotId: robotId,
+            localizationPose: localizationPose,
+            deckId: deck,
```

Comment:
why is deckId named deck? is not deckId a better name?

Reply:
deck is a function argument, but I see your point.